cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -159,7 +160,10 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
     ).convert { name ->
         allPackageManagersByName[name]
             ?: throw BadParameterValue("Package managers must be one or more of ${allPackageManagersByName.keys}.")
-    }.split(",")
+    }.split(",").deprecated(
+        message = "--package-managers is deprecated, use -P ort.analyzer.enabledPackageManagers=[] instead.",
+        tagValue = "use -P ort.analyzer.enabledPackageManagers=[] instead."
```

Comment:
I haven't checked, but should this sentence start capitalized to look good in the CLI help?

Reply:
This it what it looks like:
```
  -m, --package-managers VALUE     The comma-separated package managers to
                                   activate, any of [Bower, Bundler, Cargo,
                                   Carthage, CocoaPods, Composer, Conan,
                                   DotNet, GoDep, GoMod, Gradle, Maven, NPM,
                                   NuGet, PIP, Pipenv, Pub, SBT,
                                   SpdxDocumentFile, Stack, Unmanaged, Yarn,
                                   Yarn2]. Note that deactivation overrides
                                   activation. If set, package manager
                                   activation settings from configuration
                                   files are ignored. (deprecated: use -P
                                   ort.analyzer.enabledPackageManagers=[] on
                                   the ort command instead)
```